src/libs/actions/Session/index.js

```diff
@@ -292,29 +292,33 @@ function resetPassword() {
     {
         optimisticData: [
             {
-                onyxMethod: 'merge',
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
                 key: ONYXKEYS.ACCOUNT,
                 value: {
                     isLoading: true,
                     forgotPassword: true,
+                    message: null,
+                    errors: null,
                 },
             },
         ],
         successData: [
             {
-                onyxMethod: 'merge',
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
                 key: ONYXKEYS.ACCOUNT,
                 value: {
                     isLoading: false,
+                    message: Localize.translateLocal('resendValidationForm.linkHasBeenResent'),
```

Comment:
Hi ✋ coming from https://github.com/Expensify/App/issues/17216, where this message is not translated when a user changes the language. 

Localized text saved in Onyx is not translated when the locale is changed. So instead, we should save the key and translate it at the component level.